force-app/main/default/lwc/recentSessions/recentSessions.html

```diff
@@ -0,0 +1,122 @@
+                                    <lightning-layout-item
+                                        size="4"
+                                        class="slds-list_horizontal slds-grid_vertical-align-center"
+                                    >
+                                        <lightning-icon
+                                            tabindex="0"
+                                            icon-name="utility:success"
+                                            alternative-text={labels.success}
+                                            size="x-small"
+                                            variant="success"
+                                            title={labels.success}
+                                            if:true={session.showCompleteIcon}
+                                        ></lightning-icon>
+                                        <lightning-output-field
+                                            field-name={fields.status}
+                                            variant="label-hidden"
+                                            class="slds-var-p-left_x-small"
+                                        >
+                                        </lightning-output-field>
+                                    </lightning-layout-item>
+                                    <lightning-layout-item
+                                        size="12"
+                                        class="slds-var-p-left_x-small slds-var-m-top_xxx-small slds-list_horizontal"
+                                    >
+                                        <lightning-formatted-date-time
+                                            value={session.sessionStartDate}
+                                            hour="2-digit"
+                                            minute="2-digit"
+                                        ></lightning-formatted-date-time>
+                                        <span
+                                            class="slds-var-p-left_x-small slds-var-p-right_x-small"
+                                            >|</span
+                                        >
+                                        <span>{session.serviceName}</span>
+                                    </lightning-layout-item>
+
+                                    <lightning-layout-item
+                                        size="8"
+                                        class="slds-list_horizontal slds-var-m-top_x-small slds-var-p-left_x-small slds-grid_vertical-align-center"
+                                    >
+                                        <lightning-icon
+                                            tabindex="0"
+                                            icon-name="utility:user"
+                                            alternative-text={labels.user}
+                                            size="x-small"
+                                            title={labels.user}
```

Comment:
Is this really a contact an not a user, do we need the title either?

Reply:
It is a contact but the icon is named user so I was trying to be consistent. But we can use Contact too.